### DISCUSSION TITLE
Is it possible to show multiple experiments stored in different folders?

### DISCUSSION THREAD

**Discussion by Jingnan-Jia** [mlflow/mlflow#5827] [Q&A] [answered]
Hi, I am runing multiple different projects and want to track them. The structures are like：

Project1
- docs
- source_code
  - main.py
  - mlruns
    - 0
      - a6de16f6cb134095b607967b20eff70e
      - a6de16f6cb134095b607967b20eff70f
    - 1 

Project2
- docs
- source_code
  - main.py
  - mlruns
    - 0
      - b6de16f6cb134095b607967b20eff70e
      - c6de16f6cb134095b607967b20eff70f
    - 1 

At present, I can not see all the experiments and runs in the two different projects from mlflow UI at the same time. I have to navigate to Project1/source_code and run `mlflow ui` to see the experiments and runs for Project1, then navigate to Project2/source_code and run `mlflow ui` again to see the experiments and runs for Project2. 

However, I want to see that all the experiments and runs in the two different projects can be shown in mlflow UI at the same time. How to achieve it?

**Comment by harupy** [comment, marked as answer]
Duplicate of https://github.com/mlflow/mlflow/issues/5828
